ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransactionSender.java

```diff
@@ -38,6 +38,7 @@ public void onTransactionsAdded(final Iterable<Transaction> transactions) {
     ethContext
         .getEthPeers()
         .streamAvailablePeers()
+        .filter(transactionTracker::isPeerSupported)
```

Comment:
```suggestion
        .filter(peer -> transactionTracker.isPeerSupported(peer, EthProtocol.ETH64)
```
May require an import

Reply:
I'll commit your first suggestion and work my way from there, no worries.